shell/platform/tizen/public/flutter_tizen.h

```diff
@@ -158,11 +158,14 @@ FLUTTER_EXPORT FlutterDesktopViewRef FlutterDesktopViewCreateFromElmParent(
 // @warning This API is a work-in-progress and may change.
 FLUTTER_EXPORT void FlutterDesktopViewDestroy(FlutterDesktopViewRef view);
 
-// Returns a handle to evas object that the FlutterView is drawn to.
+// Returns a handle that the FlutterView is drawn to.
```

Comment:
How about the expression below?
```suggestion
// Returns a native UI toolkit handle for manipulation in host application.
```